src/app/components/home-carousel/index.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
+import classnames from 'classnames';
+import transitionOnScroll from 'app/lib/transition-on-scroll';
+
+import ScrollWrapper from 'app/components/scroll-wrapper';
+import TimerUI from 'app/components/timer-ui';
+import Video from 'app/components/video';
+
+const itemsRefreshInterval = 8000;
+const tickerFrequency = 50;
+const transitionDuration = 300;
+const distance = 60;
+
+function goToNextItems(component) {
+  component.setState({ tick: itemsRefreshInterval });
+
+  if (component.state.currentStartItem === component.props.carouselItems.length - 2) {
```

Comment:
Is `2` (here and in L20) a config variable referring to the no. of items in the carousel? Really minor but it would be good to extract that into a variable for clarity.